### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by cenkakin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,73 @@
 
 This is a simple Spring application that can function as a remote Maven cache server.
```
Comment (nburkley) [inline README.md:3]: Is it worth a quick sentence or two about what Maven Cache is? Or maybe we can just link to the blog post once it's published.

Comment (nburkley) [reply README.md:3]: I think it's worth including a bit more info about the caching options here, something like: "This project enables using a Maven cache to cache files locally or remotely in AWS S3"

File: README.md [hunk 2]
```diff
@@ -2,6 +2,73 @@
...
+
+    - Run the sample docker-compose:
+      ```bash
+      docker-compose -f samples/remote-cache-starters/maven-cache-server-docker-compose.yml up
+      ```
+
+    - Run maven build command with the remote cache enabled:
+      ```bash
+      cd samples/hi-world && ./mvnw clean install -s settings.xml -Dmaven.build.cache.remote.enabled=true -Dmaven.build.cache.remote.save.enabled=true && cd ../..
+      ```
```
Comment (nburkley) [inline README.md:74]: Example application - I see there's an example app embedded - is it worth a few sentences on how that works, e.g., changing the test file won't require rebuilding the controller

File: README.md [hunk 3]
```diff
@@ -2,6 +2,73 @@
...
+- Or open your IDE, and run/debug `com.github.cenkakin.mavencacheserver.MavenCacheApplication`
+
+- The server starts on port 8080
+
+Those commands use the default configuration. You can modify them
+in [application.yml](src/main/resources/application.yml) file.
+
+## How to test a remote cache server with the sample project locally
+
+- Run the remote cache server via using either:
```
Comment (nburkley) [inline README.md:62]: Run the remote cache server ~via~ using either:

**Review by nburkley — COMMENTED** [review]
Great readme!

File: README.md (hunk 1 above)
Comment (cenkakin) [reply README.md:3]: > Is it worth a quick sentence or two about what Maven Cache is? Or maybe we can just link to the blog post once it's published.

Will add the link here once it's published 👍 

> I think it's worth including a bit more info about the caching options here, something like: "This project enables using a Maven cache to cache files locally or remotely in AWS S3"

👍 will add it

**Pull request merged**